.github/workflows/ci-core-reusable.yml

```diff
@@ -323,6 +323,11 @@ jobs:
           --server-db-name=zksync_server_localhost_consensus \
           --chain consensus
 
+      - name: Export chain list to environment variable
+        run: |
+          CHAINS="era,validium,custom_token,consensus"
+          echo "CHAINS=$CHAINS" >> $GITHUB_ENV
+
```

Comment:
Tentatively addressed in https://github.com/matter-labs/zksync-era/pull/3025/commits/adb1650eb1e82580ef9bab208cec0e103c249c3e

It might be a bit of an overkill, WDYT?

Reply:
Testing a compromise https://github.com/matter-labs/zksync-era/pull/3025/commits/adb1650eb1e82580ef9bab208cec0e103c249c3e

Reply:
Yeah, agree it's too much hassle, let's revert these changes

Reply:
Reverted in https://github.com/matter-labs/zksync-era/pull/3025/commits/eac5827b4a1c9ff016c4775a15c8827f1310c656